compact/tracking/mpgd_barrel.xml

```diff
@@ -98,8 +119,16 @@
 
   <readouts>
     <readout name="MPGDBarrelHits">
-      <segmentation type="CartesianGridXY" grid_size_x="0.150*mm*sqrt(12)" grid_size_y="0.150*mm*sqrt(12)" />
-      <id>system:8,layer:4,module:12,sensor:2,x:32:-14,y:-18</id>
+      <comment> 768 strips in phi yield a segmentation of 2pi/8/768 ~= 1mrad. Which in turn, yields ~160 um in resolution.
+      </comment>
+      <segmentation type="MultiSegmentation" key="sensor">
+        <comment> In preparation for a 2-sensors-w/-strip setup. Presently sensors w/ odd key values do not exist. </comment>
+        <segmentation name="InnerPhi" type="CylindricalGridPhiZ" key_value="0" grid_size_phi="1*mrad" grid_size_z="0.150*mm*sqrt(12)" radius="MMInnerSensor_R" />
+        <segmentation name="InnerZ"   type="CylindricalGridPhiZ" key_value="1" grid_size_phi="1*mrad" grid_size_z="0.150*mm*sqrt(12)" radius="MMInnerSensor_R" />
+        <segmentation name="OuterPhi" type="CylindricalGridPhiZ" key_value="2" grid_size_phi="1*mrad" grid_size_z="0.150*mm*sqrt(12)" radius="MMOuterSensor_R" />
+        <segmentation name="OuterZ"   type="CylindricalGridPhiZ" key_value="3" grid_size_phi="1*mrad" grid_size_z="0.150*mm*sqrt(12)" radius="MMOuterSensor_R" />
+      </segmentation>
+      <id>system:8,layer:4,module:12,sensor:2,phi:32:-16,z:-16</id>
```

Comment:
Does the 16 bits in `z` still capture the full extent of what was 18 bits in `y` with the same grid size?

Reply:
I think so. My experience with this parameter is that when it is set too tight, the processing aborts with an error. Why did I choose 16 in phi and 16 in Z? Because our tiles are approximately square in size. => phi and Z play tend to play a symmetric role. Also, it's a first guess, the geometry of the tiles is not yet finalized: it's some kind agnostic first guess.

Reply:
At 16 bits and 0.150*mm*sqrt(12) grid size, the maximum extent is 34 m, so that should be fine.